source/administration/devops-command-center.rst

```diff
@@ -3,199 +3,255 @@ Mattermost Incident Management
+To view past incidents of which you are a member, from the desktop or browser:
+1. Navigate to **Main Menu > Playbooks & Incidents**.
+2. Select the **Incidents** tab.
+3. Find the incident to be reviewed and click the name of the incident.
+4. Review the incident details:
+   * The duration, total number of members ever involved, and messages sent in the channel are listed.
+   * A graphical timeline shows how much time elapsed between completing tasks.
+5. Optionally export the contents of the incident channel for review during a post-mortem.
+
+Built-in slash commands
+-----------------------
 
-To view information about ongoing incidents, select the **Incidents** icon in the header of any channel to open the RHS where all ongoing incidents are listed. Select **Go to Incident Channel** to open the relevant channel and see:
+The ``/incident`` slash command allows interacting with incidents via the post textbox on desktop, browser, and mobile:
 
-- The incident commander
-- The current stage
-- The remaining tasks
-- The finished tasks
+- ``/incident start`` - Start a new incident.
+- ``/incident end`` - End an ongoing incident.
+- ``/incident restart`` - Restart an ended incident.
+- ``/incident check [stage #] [task #]`` - Check/uncheck the specified stage's task.
+- ``/incident announce ~channel`` - Announce the current incident in other channels.
+- ``/incident list`` - List all your incidents.
+- ``/incident commander [@username]`` - Show or change the current commander.
+- ``/incident info`` - Show a summary of the current incident.
+- ``/incident stage [next/prev]`` - Move to the next or previous stage.
 
-You can also:
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
 
-- Assign a step to yourself or another incident member.
-- Mark a step as **Complete** or **Incomplete**.
-- Start an automated action.
-- Invite new members to the channel.
+To quickly Mattermost Incident Management, use the following test commands to create incidents populated with random data:
+- ``/incident test create-incident [playbook ID] [timestamp] [incident name]`` - Provide the ID of an existing playbook to which the current user has access, a timestamp, and an incident name. The command creates an ongoing incident with the creation date set to the specified timestamp.
+  * An example command looks like: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``
+- ``/incident test bulk-data [ongoing] [ended] [start date] [end date] [seed]`` - Provide a number of ongoing and ended incidents, a start and end date, and an optional random seed. The command creates the given number of ongoing and ended incidents, with creation dates randomly between the start and end dates. The seed may be used to reproduce the same outcome on multiple invocations. Incident names are generated randomly. 
```

Comment:
```suggestion
- ``/incident test bulk-data [ongoing] [ended] [start date] [end date] [seed]`` - Provide a number of ongoing and ended incidents, a start and end date, and an optional random seed. The command creates the specified number of ongoing and ended incidents, with creation dates set randomly between the specified start and end dates. The seed may be used to reproduce the same outcome on multiple invocations. Incident names are generated randomly. 
```

Reply:
Is there value in specifying where seed values come from, where users can find/use them?

Reply:
Good question. I'll defer to @agarciamontoro on that.

Reply:
Seed values are simply a user-specified number, they don't have to follow any pattern whatsoever. The gist of it is that I will get the exact same output if and only if I provide the exact same seed value (whichever it is).

Reply:
Thanks, @agarciamontoro, for this context! Can a seed value be any combination of letters and numbers? Can it be any character length? If so, great! If there are any qualifiers around what a seed value can contain, it may be worth mentioning.